src/collections/_documentation/accounts/quotas/manage-event-stream-guide.md

```diff
@@ -57,7 +57,7 @@ Once applied, you can track the filtered events (numbers and cause) using the gr
 
 ![Built-in Inbound Filters]({% asset guides/manage-event-stream/03.png @path %})
 
-## 2. Event Grouping
+## 4. Event Grouping
 
 Proper event grouping is essential for maintaining a meaningful Issue Stream and reducing redundant notifications. Sentry groups similar _Events_ into unique _Issues_ based on their _Fingerprint_. An event's fingerprint relies firstly on its **stack trace**. 
```

Comment:
"is essential to maintain a meaningful Issue Stream and reduce redundant notifications"